e2e/README.md

```diff
@@ -151,3 +151,13 @@ When using VSCode as editor, it's possible to debug the project using the existi
 In order to debug the `godog` tests, 1) you must have the `runner_test.go` file opened as the current file in the IDE, 2) Use the Run/Debug module of VSCode, and 3) select the `Godog Tests` debug configuration to be executed.
 
 ![](./debug.png)
+
+## Noticing the test framework
+
+Because we are using a project layout which consumes another directory from the same project (the CLI), and the test project uses its own `go.mod` file, totally decoupled from the CLI one, we are forced to od a workaround to generate the notice files for this project:
```

Comment:
```suggestion
Because we are using a project layout which consumes another directory from the same project (the CLI), and the test project uses its own `go.mod` file, totally decoupled from the CLI one, we are forced to do a workaround to generate the notice files for this project:
```